tests/test_conversion_testing.py

```diff
@@ -1,7 +1,14 @@
+from pandas import DataFrame
 from cdapython import Q
```

Comment:
Import DataFrame from pandas is repeated.

Reply:
i just removed it thanks